app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
```

Comment:
It's better to keep multiple imports in parentheses. It increases a redability.
Non-blocking suggestion: Imports should be presented in alphabietical order. It may be done automatically with [isort](https://pycqa.github.io/isort/) linter
```suggestion
from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
)
```